src/oldreader_api.cpp

```diff
@@ -66,24 +66,26 @@ std::string oldreader_api::retrieve_auth() {
 	}
 
 	std::string pass = cfg->get_configvalue("oldreader-password");
-    std::string pass_file = cfg->get_configvalue("oldreader-passwordfile");
-    std::string pass_eval = cfg->get_configvalue("oldreader-passwordeval");
+	std::string pass_file = cfg->get_configvalue("oldreader-passwordfile");
+	std::string pass_eval = cfg->get_configvalue("oldreader-passwordeval");
 	if ( pass == "" ) {
-        if ( pass_file != "" ) {
-            pass = read_password(pass_file.c_str());
-        } else if ( pass_eval != "") {
-            pass = eval_password(pass_eval.c_str());
-        }
+		if ( pass_file != "" ) {
+			pass = read_password(pass_file);
+			if ( pass == "" ) return "";
+		} else if ( pass_eval != "") {
+			pass = eval_password(pass_eval);
+			if ( pass == "" ) return "";
```

Comment:
Should I even change it? It's not very likely but it could break someones usage (even though I can't think of some case in which it makes sense to have a prompt if the password file doesn't exist).